src/layouts/shared/NetworkErrorBoundary/NetworkErrorBoundary.jsx

```diff
@@ -174,12 +171,26 @@ class NetworkErrorBoundary extends Component {
     // if the error is not a network error, we don't do anything and
     // another error boundary will take it from there
     if (Object.keys(errorToUI).includes(String(error.status))) {
-      sendNetworkErrorMetrics(error.status)
+      // only capture network errors if they are not a rate limit error
+      // this will typically only be schema parsing errors
+      if (error.status !== 429 && error.dev && error.error) {
```

Comment:
Thoughts on making it easier for us in the case we forget to include both `error.dev` and `error.errror` and that means our errors don't get tracked? I could definitely see myself doing that. A fallback catchall fingerprint name perhaps?

Reply:
When you use the `NetworkErrorObject` it will enforce you to add `dev` to the rejection object.

Reply:
The only issue with having a catchall is that we're likely to catch errors that are out of our control, such as repo not activated, etc. which we don't want to be flooding Sentry with.